example_test.go

```diff
@@ -694,6 +694,27 @@ func ExampleWithGlobalJobOptions() {
 	// [tag4 tag5 tag6]
 }
 
+func ExampleWithIdentifier() {
+	s, _ := NewScheduler()
+	defer func() { _ = s.Shutdown() }()
+
+	j, _ := s.NewJob(
+		DurationJob(
+			time.Second,
+		),
+		NewTask(
+			func(one string, two int) {
+				fmt.Printf("%s, %d", one, two)
+			},
+			"one", 2,
+		),
+		WithIdentifier(uuid.MustParse("87b95dfc-3e71-11ef-9454-0242ac120002")),
+	)
+	fmt.Println(j.ID())
+	// Output:
+	// [tag1 tag2 tag3]
```

Comment:
This is going to print the uuid, so I think it should be - just have to make sure that's how uuid's print 😄 
```suggestion
	// 87b95dfc-3e71-11ef-9454-0242ac120002
```

Reply:
Yes should be that
```
--- FAIL: ExampleWithIdentifier (0.00s)
got:
87b95dfc-3e71-11ef-9454-0242ac120002
want:
[tag1 tag2 tag3]
```

Reply:
@pcfreak30 fixed, i think.